### ISSUE TITLE
What is the meaning of transforms.Normalize((0.1307,), (0.3081,)) in mnist

### ISSUE THREAD

**Issue by copyrightly** [pytorch/examples#653]
In mnist/main.py, when reading the dataset using DataLoader, there is a line:

`transforms.Normalize((0.1307,), (0.3081,))`

can any one explain its meaning? I know that it tries to normalize the data, but why there are two parameters and where do those 0.1307 and 0.3081 come from?

**Comment by zhiqwang** [comment]
May be computed across all the training image of MNIST?

**Comment by xinpingwang** [comment]
see this discussion for more details, https://discuss.pytorch.org/t/normalization-in-the-mnist-example/457

**Comment by AliZandesh** [comment]
The values 0.13070.1307 and 0.30810.3081 used in the normalization process for the MNIST dataset are significant because they represent the mean and standard deviation of the pixel values across the dataset.
Normalized value = Pixel value – Mean / Standard Deviation
